packages/shared/src/react/contexts.tsx

```diff
@@ -71,7 +71,13 @@ function useAssertWrappedByClerkProvider(displayNameOrFn: string | (() => void))
     }
 
     throw new Error(
-      `${displayNameOrFn} can only be used within the <ClerkProvider /> component. Learn more: https://clerk.com/docs/components/clerk-provider`,
+      `Error: ${displayNameOrFn} can only be used within the <ClerkProvider /> component.
```

Comment:
I think an error object will prefix with `Error:` by default? In any case, I don't think we need it.
```suggestion
      `${displayNameOrFn} can only be used within the <ClerkProvider /> component.
```